dcp_prototype/backend/browser/api/app.py

```diff
@@ -0,0 +1,151 @@
+import os
+import sys
+
+import chalice
+from chalice import Chalice
+
+pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), "chalicelib"))  # noqa
+sys.path.insert(0, pkg_root)  # noqa
+
+from browser.rds.browser_orm import (DBSessionMaker,
+                                     Project, File,
+                                     LibraryPrepProtocol, Tissue, Species,
+                                     LibraryPrepProtocolJoinProject, TissueJoinProject,
+                                     SpeciesJoinProject)
+
+app = Chalice(app_name='browser-api')
+
+
+@app.route('/')
+def index():
+    return "render documentation here"
```

Comment:
Whats supposed to be here instead?

Reply:
[OpenAPI documentation page](https://app.zenhub.com/workspaces/dcp2-5e2a191dad828d52cc78b028/issues/humancellatlas/dcp-prototype/129)